src/react/endpoint/EndpointList.tsx

```diff
@@ -38,28 +34,35 @@ type Props = {
 
 function EndpointList({ endpoints, locations }: Props) {
   const dispatch = useDispatch();
-  const listRef = useRef(null);
   const showDelete = useSelector(
     (state: AppState) => state.uiEndpoints.showDelete,
   );
 
-  const handleDeleteApprove = () => {
-    if (!showDelete) {
-      return;
-    }
-
-    dispatch(deleteEndpoint(showDelete));
-  };
-
-  const handleDeleteCancel = () => {
-    dispatch(closeEndpointDeleteDialog());
-  };
+  const [hostnameFilter, setHostNameFilter] = useState();
```

Comment:
Nit: just for consistency 
```suggestion
  const [hostnameFilter, setHostnameFilter] = useState();
```